spotify_player/src/state/ui/mod.rs

```diff
@@ -83,6 +83,11 @@ impl UIState {
         match self.popup {
             Some(PopupState::Search { ref query }) => {
                 let query = query.to_lowercase();
+
+                #[cfg(feature = "fzf")]
+                return fuzzy_search_items(items, &query);
```

Comment:
```suggestion
               fuzzy_search_items(items, &query)
```

`return` is redundant here

Reply:
I thought this too but it won't let me compile.
```
error: expected `;`, found `#`
  --> spotify_player/src/state/ui/mod.rs:88:50
   |
88 |                 fuzzy_search_items(items, &query)
   |                                                  ^ help: add `;` here
89 |
90 |                 #[cfg(not(feature = "fzf"))]
   |                 - unexpected token
```

Reply:
right, I forgot